core/src/G3Timestream.cxx

```diff
@@ -529,12 +529,6 @@ G3Timestream G3Timestream::operator -(double r) const
 	return ret;
 }
 
-static
-G3Timestream operator -(double l, const G3Timestream &r)
-{
-	return (r-l)*(-1);
```

Comment:
This particular implementation does seem a bit dubious; if this function is brought back I would change it to use an explicit loop like the division function so that it makes only one pass over the data.